h3ronpy/pyproject.toml

```diff
@@ -37,6 +37,7 @@ polars = [
 ]
 pandas = [
     "geopandas>=0.10",
+    "fiona>=1.9"
```

Comment:
FYI https://github.com/geopandas/pyogrio is vectorized and much, much faster than fiona. It's the default for GeoPandas >= 1.0 but can be used opt-in before that as well.

Reply:
Also, the failing test is that you either need to upgrade geopandas or require fiona `<1.10` https://stackoverflow.com/questions/78949093/how-to-resolve-attributeerror-module-fiona-has-no-attribute-path

Reply:
Thanks you. I am mostly trying to get this to build and to release a patch version with as little breaks as possible. So I think I will go now with the fiona downgrade. After that its is time to work on a new minor-release with the other issues, a few dependency upgrades and the requirement of python >= 3.9